src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -165,8 +165,10 @@ class FileSpec private constructor(builder: FileSpec.Builder) {
   }
 
   class Builder internal constructor(
-      internal val packageName: String,
+      packageName: String,
       internal val name: String) {
+    internal val packageName = packageName.split('.')
+        .joinToString(".") { escapeIfKeyword(it) }
```

Comment:
I think we only want to do this inside of `writeTo`, since currently we'd expose this escaped version which might break someone using it for something else.